pkgs/tools/misc/empty/default.nix

```diff
@@ -0,0 +1,49 @@
+{ fetchgit, lib, stdenv, which }:
+
+let
+  baseName = "empty";
+  rev = "6e81f477dba6c2f9086da70b02abec71b64fb16e";
+  baseVersion = "0.6.21b";
+  version = baseVersion + "+1-git-" + (builtins.substring 0 7 rev);
```

Comment:
Is that an actual release or some commit after a release?

Reply:
It's a commit after a release: the release is 0.6.21b, the +1 is one commit after the release, the git-rev substring is the commit hash of that commit. Should I label it in a different way?

Reply:
We generally use upstream release numbers minus the v or the unstable prefix with the date of the commit.

Reply:
Ok: as the single additional commit is not critical (especially in a packaged context), I'll package the version.